### DISCUSSION TITLE
compat with asdf version manager

### DISCUSSION THREAD

**Discussion by dimonnwc3** [Homebrew/discussions#1057] [Everyday usage]
brew shows a warning when  using with `asdf` version manager

I created an issue in `asdf` repo with a detailed description: https://github.com/asdf-vm/asdf/issues/894

some suggestion on how to fix it is welcome

**Comment by carlocab** [comment]
The warning says this:
```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!
```
If you're not experiencing any problems, just ignore the warning.
